os/board/common/Kconfig

```diff
@@ -106,3 +106,27 @@ config SECOND_FLASH_PART_NAME
 
 endif # SECOND_FLASH_PARTITION
 endif # ARCH_BOARD_HAVE_SECOND_FLASH && (NFILE_DESCRIPTORS != 0)
+
+config AUTOMOUNT
+	bool "Automount partitions"
+	default n
+	depends on FLASH_PARTITION
+	depends on !DISABLE_MOUNTPOINT
+	---help---
+		If enabled, mount userfs or romfs partition automatically at boot.
+
+config AUTOMOUNT_USERFS
+	bool "Automount userfs partiton"
+	default n
+	depends on AUTOMOUNT
+	depends on FS_SMARTFS
+	---help---
+		If enabled, mount userfs partition automatically at boot.
+
+config AUTOMOUNT_ROMFS
+	bool "Automount romfs partiton"
+	default y
```

Comment:
Is there any reason to be y even AUTOMOUNT_USERFS is defaultly n?

Reply:
Changed. Thanks.